lib/tasks/cucumber.rake

```diff
@@ -1,18 +1,42 @@
-require "cucumber/rake/task"
+# IMPORTANT: This file is generated by cucumber-rails - edit at your own peril.
+# It is recommended to regenerate this file in the future when you upgrade to a
+# newer version of cucumber-rails. Consider adding your own code to a new file
```

Comment:
It looks as though the best approach would be to just generate this file from cucumber-rails (a sort of a return to 'factory settings'), rather than try to edit it, as 6a13dbf52 did that and caused problems.

The original problem 6a13dbf52 was trying to solve was to fix linting errors - but we could add a rule to [.rubocop.yml](https://github.com/alphagov/collections/blob/master/.rubocop.yml) to exclude `lib/tasks/cucumber.rake`, which seems reasonable for a generated file.

Reply:
I think that's sensible @ChrisBAshton 👍
@benthorner and @hannako what do you think?

Reply:
Can do. I've also got this, since we're unlikely to ever need to regenerate this file: https://github.com/alphagov/collections/pull/1683.

Reply:
👋 @ChrisBAshton  I've regenerated the files.  I'll squash if you're happy?

Reply:
Ah, I'll rebase everything

Reply:
Just posting here, since we merged https://github.com/alphagov/collections/pull/1683, which should fix the problem.

Reply:
We could use this one if we want to keep the original file. I've always found it surprising to see such a complicated rake task with all the other ones, but I appreciate that's more personal preference.

Reply:
Yeah, I'll close this, thanks @benthorner